src/lib/mode.ts

```diff
@@ -40,7 +41,7 @@ export function setInitialMode(defaultMode: Mode, themeColors?: ThemeColors) {
 		}
 	}
 
-	localStorage.setItem('mode', mode);
+	localStorage.setItem('mode-watcher-mode', mode);
```

Comment:
Let's use the exported const variable so we don't have to worry about this in the future.

Reply:
@huntabyte I've definitely tried using the variable before and that did not work, since the function is stringified and then the reference to that varible does not work anymore. I can try again just to be 100% sure

Reply:
with this change:
```diff
diff --git a/src/lib/mode.ts b/src/lib/mode.ts
index 68d9eb4..9aecab2 100644
--- a/src/lib/mode.ts
+++ b/src/lib/mode.ts
@@ -26,7 +26,7 @@ export function resetMode(): void {
 /** Used to set the mode on initial page load to prevent FOUC */
 export function setInitialMode(defaultMode: Mode, themeColors?: ThemeColors) {
 	const rootEl = document.documentElement;
-	const mode = localStorage.getItem('mode-watcher-mode') || defaultMode;
+	const mode = localStorage.getItem(localStorageKey) || defaultMode;
 	const light =
 		mode === 'light' ||
 		(mode === 'system' && window.matchMedia('(prefers-color-scheme: light)').matches);
@@ -41,7 +41,7 @@ export function setInitialMode(defaultMode: Mode, themeColors?: ThemeColors) {
 		}
 	}
 
-	localStorage.setItem('mode-watcher-mode', mode);
+	localStorage.setItem(localStorageKey, mode);
 }
 
 export { localStorageKey, userPrefersMode, systemPrefersMode, derivedMode as mode, themeColors };
```
I get the following error in the console after running `pnpm build && pnpm preview`:
```
[Error] ReferenceError: Can't find variable: localStorageKey
	setInitialMode (localhost:11)
	Global Code (localhost:22)
```

Reply:
Interesting... we'll investigate this further. In the meantime, I'll merge!